library/src/main/java/com/novoda/downloadmanager/DownloadBatch.java

```diff
@@ -54,10 +55,10 @@ void setCallback(DownloadBatchStatusCallback callback) {
 
     void download() {
         String rawBatchId = downloadBatchStatus.getDownloadBatchId().rawId();
-        Log.v("start download " + rawBatchId + ", status: " + downloadBatchStatus.status());
+        Log.v("start sync download " + rawBatchId + ", " + STATUS + " " + downloadBatchStatus.status());
```

Comment:
FYI for log I think you can just do `,` instead of concatenating with `+`

Reply:
Yeah, with NoTils... but also with the new logger, so no problem 👍